src/elements/SelectInput.vue

```diff
@@ -29,28 +29,41 @@ const onInvalid = (evt: HTMLInputElementEvent) => {
   <label class="wrapper" :for="name">
     <span class="label">
       <slot />
-      <span v-if="required && (model === null || model === '')" class="required">*</span>
+      <span v-if="required && (model === null || model === '')" class="required"
+        >*</span
+      >
```

Comment:
Can we wrap code at 120? I thought we had a rule for this, but I might be wrong and that's just my personal settings 😅